frontend/src/components/Sidepanel/Sidepanel.tsx

```diff
@@ -77,7 +85,10 @@ const Sidepanel = () => {
         {panels.map(panel => (
           <SidebarButton
             key={panel.value}
-            onClick={() => setActivePanel(activePanel?.value === panel.value ? undefined : panel)}
+            onClick={() => {
+              setActivePanel(activePanel?.value === panel.value ? undefined : panel)
+              stopTemplateInsert(setTemplate, setTool)
+            }}
```

Comment:
Make it so that template insert mode stops when you switch to another sidepanel, before it was only when you close side panel